src/lib.rs

```diff
@@ -72,7 +72,7 @@
 #[cfg(test)]
 mod test;
 
-// The guts module is for incremental use cases like the `bao` crate that need
```

Comment:
I think "need" is actually correct here, since the subject of the clause is "cases".